docs/src/pages/[platform]/components/heading/demo.tsx

```diff
@@ -1,21 +1,42 @@
 import * as React from 'react';
-import { Heading, HeadingLevel } from '@aws-amplify/ui-react';
+import { Heading, HeadingLevel, HeadingProps } from '@aws-amplify/ui-react';
 
 import { HeadingPropControls } from './HeadingPropControls';
 import { Demo } from '@/components/Demo';
 
-const propsToCode = (level: HeadingLevel) => {
-  return `<Heading level={${level}}>Heading</Heading>`;
+const propsToCode = (
+  level: HeadingLevel,
+  isTruncated: boolean,
+  value: string
+) => {
+  return `<Heading
+  width='30vw'
+  level={${level}} ${isTruncated ? `\n  isTruncated={true}` : ''}
+>
+   ${value}
+</Heading>`;
 };
 export const HeadingDemo = () => {
   const [level, setLevel] = React.useState<HeadingLevel>(6);
-
```

Comment:
nit: Looks like this doc is pretty old and do not use the props map to preserve user input. But not a blocker.

Reply:
Done